dropgz/build/cni.Dockerfile

```diff
@@ -24,6 +24,7 @@ WORKDIR /dropgz
 COPY dropgz .
 COPY --from=azure-ipam /azure-ipam/*.conflist pkg/embed/fs
 COPY --from=azure-ipam /azure-ipam/azure-ipam pkg/embed/fs
+COPY --from=azure-vnet /azure-container-networking/cni/azure-$OS.conflist pkg/embed/fs/azure-swift.conflist
 COPY --from=azure-vnet /azure-container-networking/cni/azure-$OS-swift.conflist pkg/embed/fs/azure-swift.conflist
```

Comment:
can both conflists be copied to `pkg/embed/fs/azure-swift.conflist`?

Reply:
definitely not, glad you caught that